src/gui/sdlmain.cpp

```diff
@@ -2148,6 +2144,7 @@ void Disable_OS_Scaling() {
 
 //extern void UI_Init(void);
 int main(int argc, char* argv[]) {
+	int rcode = 0; // assume good until proven otherwise
```

Comment:
Might as well add proper return code handling if we hit any exceptions (`rcode` set to 1 in the `catch` statements).